content/posts/2021/pangeo-goes-live/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Pangeo goes live on 2i2c!"
+subtitle: ""
+summary: "2i2c are pleased to announce that the first Pangeo JupyterHub is now live on 2i2c-operated infrastructure! :tada: "
+authors: ["Sarah Gibson"]
+tags: [2i2c, pangeo]
+categories: [updates]
+date: 2021-11-08
+featured: false
+draft: false
+---
+
+We have recently finished re-creating the Pangeo community JupyterHub hosted on GCP in the [2i2c-org/infrastructure](https://github.com/2i2c-org/infrastructure) repository.
+This is a huge milestone in our partnership with Pangeo to provide expertise and operations of cloud-based, vendor-agnostic Jupyter infrastructure and workflows.
+
+For users of Pangeo's infrastructure, the switch should be a smooth one!
+The new hub should behave nearly identically to the old one, and will be managed by 2i2c engineers moving forward, in partnership with the Pangeo community.
+It will be available at the same URL (https://us-central1-b.gcp.pangeo.io) and your home directories were synced to the new hub only a few days before the migration took place.
+Development and operations on this hub will all be done in the open, and [we invite participation and feedback from others]() in our infrastructure work.
```

Comment:
What's an appropriate link here?

Reply:
Porting my comment from the hackmd 😉 

> Mmm... if we are requesting feedback maybe we could create some specific form? Or maybe some discourse post for getting feedback?

Reply:
Hmm, maybe this one? https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890

Reply:
```suggestion
Development and operations on this hub will all be done in the open, and [we invite participation and feedback from others](https://discourse.pangeo.io/t/migration-of-us-central1-b-gcp-pangeo-io-to-2i2c-infrastructure/1890) in our infrastructure work.
```

Reply:
I am unresolving this after a Slack discussion yesterday

> we should also be encouraging the Pangeo community to open support tickets as well 

**cc:** @choldgraf

Reply:
I opened up an issue to discuss this here: https://github.com/2i2c-org/team-compass/issues/297 

I think our goal should be to develop a prototype for what this kind of support might be, and try it out with the the Pangeo community to see if it feels sustainable. Does that make sense?